figures/admin.py

```diff
@@ -85,3 +85,14 @@ class PipelineErrorAdmin(admin.ModelAdmin):
         ('course_id', AllValuesDropdownFilter),
         ('user', RelatedOnlyFieldListFilter),
         'error_type')
+
+
+@admin.register(figures.models.CourseMauMetrics)
+class CourseMauMetricsAdmin(admin.ModelAdmin):
+    """Defines the admin interface for the CourseMauMetrics model
+    """
+    list_display = ('id', 'date_for', 'site', 'course_id', 'mau')
+    list_filter = (
+        ('site', RelatedOnlyDropdownFilter),
+        ('course_id', AllValuesDropdownFilter),
```

Comment:
That'll be a bit slow, no?

Reply:
It is slow when there are are a lot of records, but at least it is a starting point